### ISSUE TITLE
[A11y] 2.4.7 Focus Visible – Visible Tab Focus Indicator Missing On Map

### ISSUE THREAD

**Issue by bsaadat** [publichealthengland/coronavirus-dashboard#77]
Tester has observed that when tabbing the page, a visible tab focus indicator is not present the second time the user tabs to the map. It is expected that a visible indicator is present to highlight the element in focus for any keyboard operable user interface components. This can be the default browser indicator but it must not prevent users from applying their own custom focus styles.
 
**This does not comply with WCAG Level A 2.1:
2.4.7 Focus Visible** - https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html 
 
**Steps to Recreate:**
1.    Navigate to URL
2.    Tab the page content until map is reached
3.    Observe issue
 
**This has been observed on the following pages:**
Coronavirus (COVID-19) cases in the UK
 
**Environment:**
Windows 10 – Chrome 80
 
**This issue is known to occur on:**
Believed to be global
 
**Reproduction rate:**
100% - Consistently reproducible. Occurs every time that it is attempted.
 
**Supporting material:**
Screenshot/Video
 
**Version:**
https://coronavirus.data.gov.uk/
 
3.Minor Impact on Functionality

**Comment by bsaadat** [comment]
![focus_indicator](https://user-images.githubusercontent.com/27902577/80503600-00bac400-896a-11ea-8308-aa8dae190fe7.gif)

**Comment by bsaadat** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[A11y] 2.4.7 Focus Visible – Visible Tab Focus Indicator Missing On Map](https://trello.com/c/k5rHGWNU/223-a11y-247-focus-visible-visible-tab-focus-indicator-missing-on-map)
